config/blobs.yml

```diff
@@ -46,10 +46,10 @@ docker/autoconf-2.69.tar.gz:
   size: 1927468
   object_id: 83391b7e-d4bf-4052-92bb-e708f33e25c6
   sha: 562471cbcb0dd0fa42a76665acf0dbb68479b78a
-docker/docker-17.06.1.tgz:
-  size: 29699281
-  object_id: 128671d4-33ef-4cad-8239-0ae39d38f55d
-  sha: 01c73e6548e588e6d5bdc3dea45bb90239516e4b
+docker/docker-18.09.2.tgz:
```

Comment:
Is this the latest one or can we update to an even higher version ?

Reply:
Latest version available is 18.09.6 (https://download.docker.com/linux/static/stable/x86_64/)

I have updated it to 18.09.2 as per the details in the ticket

Reply:
ok...should be fine